Telereso/Classes/Telereso.swift

```diff
@@ -16,20 +17,15 @@ public struct Telereso{
     static private var isStringLogEnabled = false
     static private var isDrawableLogEnabled = false
     static private var _isRealTimeChangesEnabled = false
-    
-    //    private var listenersList = hashSetOf<RemoteChanges>()
     static private var stringsMap = [String : [String : JSON]]()
+    static private var drawablesMap = [String : [String : JSON]]()
     static private var currentLocal: String?
-    //    private var drawableMap = HashMap<String, JSONObject>()
-    //    private var densityList = listOf("ldpi", "mdpi", "hdpi", "xhdpi", "xxhdpi", "xxxhdpi")
-    //
-    
     static private var remoteConfigSettings: RemoteConfigSettings?
     static private var remoteConfig: RemoteConfig!
-        
-    static public func initialize(completionHandler: (() -> Void)? = nil){
+
+    static public func initialize(locale: String? = nil, completionHandler: (() -> Void)? = nil) {
```

Comment:
do we need another function ? 

if we can do the following examples  using only this function, then we are good 

```swift
initialize()
initialize{
// logic
}

initialize("en")
initialize("en"){
// logic
}
```